editor/import/editor_scene_importer_gltf.cpp

```diff
@@ -986,7 +986,7 @@ Error EditorSceneImporterGLTF::_parse_meshes(GLTFState &state) {
 				Ref<SurfaceTool> st;
 				st.instance();
 				st->create_from_triangle_arrays(array);
-				if (p.has("targets")) {
+				if (!p.has("targets")) {
```

Comment:
Given the regression in #32712 and the comments below which specifically mention that this code should apply to morph targets (while this makes it apply to everything else), I guess this was not the right fix.

The code of this `if` block was probably breaking blend shapes, and by not using it for blend shapes the issue was fixed, but it seems problematic for other things.

Maybe what should be done is simply removing this `if` block?

Reply:
That makes a lot of sense.  I haven't tried it, but I'm curious.